src/components/CourseTable/index.jsx

```diff
@@ -33,14 +34,29 @@ const CourseTable = () => {
   }));
 
   return (
-    <TableContainer
-      id="courses"
-      className="courses"
-      fetchMethod={DiscoveryDataApiService.fetchCourses}
-      columns={courseTableColumns}
-      formatData={formatCourseData}
-      tableSortable
-    />
+    <div className="container">
+      <div className="row justify-content-md-center">
+        <div className="col-md-10">
+          <div className="row justify-content-md-end">
+            <Link to="/courses/new">
+              <button className="btn-outline-primary">New Course</button>
```

Comment:
Probably want the "btn" class too (that I think handles some common cases like disabled state)

Reply:
Oh Thanks